### DISCUSSION TITLE
A small problem with text color

### DISCUSSION THREAD

**Discussion by Allenyzh** [AutoDarkMode/Windows-Auto-Night-Mode#896] [General]
![image](https://github.com/user-attachments/assets/c24b7bae-2f53-45f2-b97b-8df8d81457d6)
When I switch themes, if I open the Windows 11 search bar immediately after clicking to switch, the text color does not change. I need to open it again for the color to update. Similarly, if the theme is black at night with white text and I shut down the computer, in the morning when I turn it on and open the search bar, the theme might appear white, but the text will also be white, causing a visibility issue.

**Comment by Mongoose6108** [comment]
I have the same problem the dark theme is all mess up including the text

**Reply by jordansmellie** [reply]
Yep, dark theme is malfunctioning for me too, specifically for the Windows taskbar. Everything else in the dark theme loads correctly every time, but after a recent update, the taskbar started consistently loading as white text on a very light gray background (or a very light version of the Accent Color, if I turn that option on). None of the troubleshooting steps listed in the Github have had any effect. I have to manually go to Apps > System > "Always light mode" then "Disabled" in order to get the taskbar usable again, but then of course all of the System items are permanently in Light Mode, which sort of defeats the purpose of this tool in the first place.

EDIT: after seeing a Reddit comment, I tried going into Task Manger and restarting Windows Explorer (explorer.exe). That fixed Dark Mode immediately, so I guess I'll just wait and see what happens with the Light>Dark automatic switch over the next few days.

**Comment by Jay-o-Way** [comment]
@Allenyzh what is your Windows version/build? And do you have the setting about the DWM checked? Might not be an ADM problem.

**Comment by Mongoose6108** [comment]
@Jay-o-Way I have the same issues , could you please tell me what option to turn off or to turn on n DWM to fix this problem with ADM?

**Comment by Jay-o-Way** [comment]
I have seen the same thing now. It's in the implementation - it's up to the window to update. I see the exact same thing when Windows itself changes theme. ADM is not to blame!
